extension/src/cli/dvc/constants.ts

```diff
@@ -0,0 +1,73 @@
+export const MIN_CLI_VERSION = '2.11.0'
+export const LATEST_TESTED_CLI_VERSION = '2.18.1'
+export const MAX_CLI_VERSION = '3'
+
+export const UNEXPECTED_ERROR_CODE = 255
+
+export enum Command {
+  ADD = 'add',
+  CHECKOUT = 'checkout',
+  COMMIT = 'commit',
+  DIFF = 'diff',
+  EXPERIMENT = 'exp',
+  INITIALIZE = 'init',
+  LIST = 'list',
+  MOVE = 'move',
+  PLOTS = 'plots',
+  PULL = 'pull',
+  PUSH = 'push',
+  REMOVE = 'remove',
+  ROOT = 'root',
+  STATUS = 'status',
+  PARAMS = 'params',
+  METRICS = 'metrics'
+}
+
+export enum SubCommand {
+  SHOW = 'show'
+}
+
+export enum Flag {
+  FORCE = '-f',
+  OUTPUT_PATH = '-o',
+  RECURSIVE = '-R',
+  SHOW_JSON = '--show-json',
```

Comment:
```suggestion
  SHOW_JSON = '--json',
```

We have softly deprecated the `--show-json` flag for the shorter `--json` flag, since 2.8.0 (see https://github.com/iterative/dvc/pull/6711, and https://github.com/iterative/dvc/releases/2.8.0).

Reply:
Has `--json` been available for all of the required commands since `2.11.0`?

Reply:
I'll have to bump the min required cli version when we start to consume `data status`. I can make the change then.

Reply:
yes, since 2.8.0, all of the commands that have `--show-json` should have `--json` alias. `plots --json` was introduced in 2.9.3 with both flags, and `data status --json` since 2.15.0.